components/webui/client/src/pages/SearchPage/SearchControls/TimeRangeInput/index.tsx

```diff
@@ -68,8 +68,12 @@ const TimeRangeInput = () => {
                 onChange={handleSelectChange}/>
             {timeRangeOption === TIME_RANGE_OPTION.CUSTOM && (
                 <DatePicker.RangePicker
+
+                    // `ShowNow` and `allowClear` options are disabled since they set the timezone
+                    //  of the range picker to the local timezone.
```

Comment:
```suggestion
                    // NOTE: We disable `showNow` and `allowClear` since they set the timezone of
                    // the range picker to the local timezone.
```